src/box/alter.cc

```diff
@@ -4299,6 +4299,12 @@ on_commit_schema_set_bootstrap_leader_uuid(struct trigger *trigger, void *event)
 	struct tt_uuid *uuid = (struct tt_uuid *)trigger->data;
 	bootstrap_leader_uuid = *uuid;
 	is_supervised_bootstrap_leader = tt_uuid_is_equal(uuid, &INSTANCE_UUID);
+	if (is_supervised_bootstrap_leader)
```

Comment:
I'd rather not make any condition here. `instance %s is assigned as a bootstrap leader` is enough, IMHO. We never say, whether UUID is our own or not in other log messages

Reply:
OK, changed:

```diff
diff --git a/src/box/alter.cc b/src/box/alter.cc
index 9eb59852a9..6db60115e2 100644
--- a/src/box/alter.cc
+++ b/src/box/alter.cc
@@ -4299,12 +4299,8 @@ on_commit_schema_set_bootstrap_leader_uuid(struct trigger *trigger, void *event)
 	struct tt_uuid *uuid = (struct tt_uuid *)trigger->data;
 	bootstrap_leader_uuid = *uuid;
 	is_supervised_bootstrap_leader = tt_uuid_is_equal(uuid, &INSTANCE_UUID);
-	if (is_supervised_bootstrap_leader)
-		say_info("instance %s (this one) is assigned as a bootstrap "
-			 "leader", tt_uuid_str(uuid));
-	else
-		say_info("instance %s is assigned as a bootstrap leader",
-			 tt_uuid_str(uuid));
+	say_info("instance %s is assigned as a bootstrap leader",
+		 tt_uuid_str(uuid));
 	box_broadcast_ballot();
 	return 0;
 }
diff --git a/test/replication-luatest/bootstrap_strategy_test.lua b/test/replication-luatest/bootstrap_strategy_test.lua
index 7ac56f12d3..1a510a42e0 100644
--- a/test/replication-luatest/bootstrap_strategy_test.lua
+++ b/test/replication-luatest/bootstrap_strategy_test.lua
@@ -871,21 +871,8 @@ g_supervised.test_early_leader_with_replica = function(cg)
     end)
     t.helpers.retrying({}, cg.server1.assert_follows_upstream, cg.server1, 1)
 
-    -- There are three related log messages:
-    --
-    -- I> this instance is assigned as a bootstrap leader
-    -- I> instance <uuid> (this one) is assigned as a bootstrap leader
-    -- I> instance <uuid> is assigned as a bootstrap leader
-    --
-    -- The first one is issued before the logger is configured, so
-    -- goes to stderr. As far, as I know, luatest has no API to
-    -- catch it.
-    --
-    -- Two others are verified below.
-
     -- Verify a log message on a bootstrap leader.
-    local exp_msg = 'instance [0-9a-f-]+ %(this one%) is assigned as a ' ..
-        'bootstrap leader'
+    local exp_msg = 'instance [0-9a-f-]+ is assigned as a bootstrap leader'
     local found = grep_log(cg.server2, exp_msg)
     t.assert(found)
     t.assert_str_contains(found, leader_uuid)
```